### DISCUSSION TITLE
Live Commerce (Sell your product through live streaming)

### DISCUSSION THREAD

**Discussion by rahul7827** [ant-media/Ant-Media-Server#6540] [Show and tell]
Hello Community members.

I am thrilled to unveil my latest WordPress plugin designed to supercharge your online store! 🎉

✨ What’s This All About? ✨

My WordPress plugin integrates seamlessly with WooCommerce and AntMedia Server, bringing you an innovative way to sell products online while engaging with your audience in real-time. It’s a game-changer for anyone looking to enhance their eCommerce experience with live interactions and sales!

🔹 Prerequisite:
- Woocommerce
- Ant Media Server

🔹 Key Features:

**Seamless Integration**: Connects effortlessly with WooCommerce and AntMedia Server.
**Live Streaming**: Go live and showcase your products in real-time.
**Interactive Chat**: Engage with viewers directly through integrated chat.
**Quick Setup**: Admins can configure settings and go live in just 5 minutes!
**Boost Sales**: Live demos with direct purchasing options to increase conversion rates.

🌟 Why You’ll Love It:

This plugin is designed for ease of use and flexibility. Whether you're running a small boutique or a large-scale online store, you’ll find it incredibly easy to set up and start broadcasting your products live. Imagine the possibilities of hosting live product launches, Q&A sessions, or exclusive sales events!

Ready to Elevate Your Online Store?

Let’s make online selling more dynamic and interactive than ever. Your customers will love the personal touch, and you’ll love the results! 🙌

Happy streaming and selling! 📈💬

#WooCommerce #LiveStreaming #eCommerce #AntMediaServer #WordPressPlugin #OnlineSelling #LiveChat #TechInnovation

**Frontend:**
![1](https://github.com/user-attachments/assets/3f7d073e-c775-4d7c-a7a1-e045d309e5b0)
![2](https://github.com/user-attachments/assets/d9471c25-9132-4457-aa76-d7cc435fe401)
![3](https://github.com/user-attachments/assets/22fe6e1a-31ec-4b76-93ce-23d0431f0445)

**Backend:**
![admin-1](https://github.com/user-attachments/assets/f7c3cbdd-adc8-4be2-8d87-bcb6071e28ea)
![admin-2](https://github.com/user-attachments/assets/3c7bf5a4-4ee6-4db6-acc0-71a07d2d0566)

**Comment by fsan2val** [comment]
Hello, I congratulate you on what you created, it will surely be a success. can you tell me the name of the plugin? do you have any demo? and what is the probability of making one for owncast? greetings

**Reply by rahul7827** [reply]
Hi @fsan2val 

Thank you so much for your kind words and congratulations!. Suggest any good name for the plugin (Currently no official name, I am considering "Ant Live Sales").

It is in my local system (Will update on server this weekend, after completing two more features of it.). If you would like to have a look you can schedule a meeting.

I am not familiar with **owncast**.  Willing to explore.

**Reply by fsan2val** [reply]
Thanks for your answer.

On the other hand, I use owncast more, I thought you had something with this one. 
It would be good if you put it in the WP repository or for sale, if you want a tester let me know so I can help you.
In the meantime I'm going to check Ant to see how it behaves.

**Reply by rahul7827** [reply]
Thank you for your suggestion, @fsan2val. I need to refactor and test the code, but I’ll share the demo URL and present the plugin in the next Ant Media webinar, if permitted. [Follow on Linkedln](https://uk.linkedin.com/company/antmedia) to get the webinar notification.

Check out Ant Media and its SDK— It is developer-friendly. If you need any assistance, feel free to reach out.

**Reply by fsan2val** [reply]
Perfect, I'll be on the lookout for the big launch. greetings

**Reply by rahul7827** [reply]
Hello @fsan2val 

I am preparing my plugin to meet WordPress plugin repo standard. Suggest the way of testing the plugin across several release of woo-commerce [(Woocommerce release).](https://developer.woocommerce.com/releases/) Doing things manually would not be easy. I found WP-CLI is a way.

Thank you
Rahul

**Reply by rahul78275** [reply]
Hello @fsan2val 

It is now available in wordpress plugin directory 

https://wordpress.org/plugins/live-sales-for-woocommerce/

**Comment by muratugureminoglu** [comment]
@rahul7827 

Congratulations! If you need any help, we're here. Keep that in mind. :)

**Reply by rahul7827** [reply]
Hello @muratugureminoglu 

Unfortunately latest work of my plugin got deleted during the development 😭 . (By mistake I uninstalled the plugin during the test and all files of the plugin got deleted.). 

Is there any way to recover it back ? (I lost a week of the my latest work).

I am using windows 11 OS, and "vs code" as editor.

-- Files are not in recycle bin.
-- I tried "CTRL + Z" in vs code, no luck.
-- Right click (properties > restore). No luck.

I tried a recovery application "Recuva", "Windows File recovery". Recuva recovered only 10% of latest work and there is corrupted data.

I can't tell you how I feel right now.

**Reply by muratugureminoglu** [reply]
Hi @rahul7827 

I have used EaseUS Data Recovery in the past, but no data had written to the disc.

**Reply by rahul7827** [reply]
I am able to recover from VS CODE history. But restoring each of the file manually was painful.

**Comment by mekya** [comment]
Hi @rahul7827,

Congratulations. 👏 

What do we need to do to test it? 

Shall we have a meeting to talk about what we can do for you? If yes, I gently ask you to schedule it here 
https://calendly.com/mekya

Cheers
Oz

**Reply by rahul7827** [reply]
Thank you @mekya .

I have scheduled meeting, Unfortunately I deleted my plugin last night. But I am able to rewrite and recover some part. So will update on the server today and will send you the demo credentials and url.

Thank you

**Comment by rahul7827** [comment]
Hello @fsan2val @mekya 

I have submitted my plugin to WordPress repository. The expected time for the review is **26 days** (Which is longer).
